.travis.yml

```diff
@@ -45,6 +45,11 @@ jobs:
       script:
         - yarn test
 
+    #- stage: 'Production Tests'
```

Comment:
Can you make an issue and make a `# TODO: Fix #1234` comment here?

Reply:
There already was one. https://github.com/ember-cli/ember-cli/issues/8154. Added a comment to this file linking the issue.